dm/worker/config_test.go

```diff
@@ -14,20 +14,49 @@
 package worker
 
 import (
+	"path"
+	"strings"
+
 	. "github.com/pingcap/check"
 )
 
-func (t *testWorker) TestConfig(c *C) {
-	cfg := &Config{}
+func (t *testServer) TestConfig(c *C) {
+	cfg := NewConfig()
 
-	err := cfg.configFromFile("./dm-worker.toml")
-	c.Assert(err, IsNil)
+	c.Assert(cfg.Parse([]string{"-config=./dm-worker.toml"}), IsNil)
```

Comment:
check more config items both in the command line and in the file?

Reply:
add one `relay-dir` and one is all